objects/_o-link-complex.scss

```diff
@@ -43,6 +43,7 @@ $o-link-complex-apply-at-breakpoints-for-target:   false !default;
   &,
   &:hover,
   &:focus {
+    border-bottom: 0;
```

Comment:
So I added that cause it's common to apply the underline via `border-bottom` instead of `text-decoration: underline` esp when you want to transition the underline (I just did it for CM's login views). I'm not a fan of overriding rules that don't exist but I decided too in this case as you shouldn't be having to write any CSS concerned with Objects in your project layer.